### DISCUSSION TITLE
the data that to the distributed training should be load all  in every node or apart? or just load the main node?

### DISCUSSION THREAD

**Discussion by mobilejammer** [bghira/SimpleTuner#1231] [Q&A] [answered]
as the title

**Comment by bghira** [comment]
they need shared storage.

**Reply by mobilejammer** [reply]
> they need shared storage.

Do you mean creating a shared directory named "datasets" on the host and then sharing it with all the child nodes? Or do you mean copying the data to each node and configuring the database directories on each node?

**Reply by bghira** [reply, marked as answer]
please follow "DISTRIBUTED.md" document.
